### ISSUE TITLE
Just a question

### ISSUE THREAD

**Issue by mbonaci** [nikospara/require-lazy#7]
I see not much activity lately. Is it because the method is obsolete or simply works and there's no need for improvements? 
If you don't mind, I would basically like to know what can I do to achieve deferred loading of JS. 
Do you still recommend using your library or you have some other suggestions (without waiting for HTTP2 so I can dispatch in parallel more than 4 requests). 
Thanks

**Comment by nikospara** [comment]
Hi, thanks for the contact.

I am still using this library whenever I can or find reason to. There are others that do similar (better? other? unrelated?) things, that I have had no time to look, unfortunately! (One example: [Webpack](http://webpack.github.io/))

There is another reason for no activity: It is perfect :) Just kidding, the truth is (given my limited time) I find no feasible ideas of improvement, but whenever I come up with some, I will find some time to implement them. The library, however, is quite usable as is.

*Any ideas you or anyone else can think would be very appreciated!*

About **deferred loading of JS**:

There are many methods, depending on what exactly do you want. This library helps you easilly define *bundles of AMD modules* and lazy load those. If you are not interested in bundling your AMD modules (to reduce HTTP calls mainly), then plain RequireJS (or - maybe - [systemjs](https://github.com/systemjs/systemjs)) can do the job. Use `require(['a', 'b', 'c'], function(a, b, c){...})` to lazy load a single module and its dependencies. Other libraries, e.g. [HeadJS](http://headjs.com/) lazy load plain JS files. Also lazy loading depends on the environment. My other project, [angular-require-lazy](https://github.com/nikospara/angular-require-lazy), is using this to lazy load Angular artifacts. I am working on some improvements on angular-require-lazy these nights...

To get a better feeling of what this library does I suggest you run and examine the examples, even the simpler ones will do, given you have some experience with AMD/RequireJS.

Feel free to contact me should you need anything else, or can come up with more details about your specific use case.

Nikos

**Comment by mbonaci** [comment]
Thanks for the great answer, I'll definitely provide feedback should I end up using your library +:100:
